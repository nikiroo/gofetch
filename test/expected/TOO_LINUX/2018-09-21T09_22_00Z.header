0Acquisition : Adimeo s’offre la société OpCoding	null/TOO_LINUX/2018-09-21T09_22_00Z		70
i  2018/09/21
i
i    OpCoding,  expert PHP Javascript, entre dans le giron d’Adimeo,
i    une  entreprise  fondée il y a 13 ans en France. Sa présence se
i    renforce   en  région  :  après  Paris,  place  à  Metz  et  au
i    Luxembourg.  Le  rachat  d’Opcoding  va  permettre  à Adimeo de
i    renforcer   ses   équipes   techniques   pour   faire  face  au
i    développement  commercial  de  la  société.  Cette  acquisition
i    va (...)
i
