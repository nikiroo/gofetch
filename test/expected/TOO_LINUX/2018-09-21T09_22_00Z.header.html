<!DOCTYPE html>
<html>
<head>
  <meta http-equiv='content-type' content='text/html; charset=utf-8'>
  <meta name='viewport' content='width=device-width, initial-scale=1.0'>
  <style type='text/css'>
    body { margin: 1em 15%; }
  </style>
</head>
<body>
<div class='story-header'>
	<h2><a href='2018-09-21T09_22_00Z.html'>Acquisition : Adimeo s’offre la société OpCoding</a></h2>
	<div class='details'>(2018/09/21)</div>
	<br/>
	<div class='content' style='text-align: justify'>
		OpCoding, expert PHP Javascript, entre dans le giron d’Adimeo, une entreprise fondée il y a 13 ans en France. Sa présence se renforce en région : après Paris, place à Metz et au Luxembourg. Le rachat d’Opcoding va permettre à Adimeo de renforcer ses équipes techniques pour faire face au développement commercial de la société. Cette acquisition va (...)
	</div>
<hr/>
</div>
</body>
